src/setup-k8s/index.ts

```diff
@@ -246,4 +256,4 @@ export default class SetupDockerGenerator extends Generator {
       hooks,
     });
   }
-}
+}
```

Comment:
missing line at the end, which should be fixed by linter - execute `npm run lint` before committing